packages/core/src/app/ui/form/DynamicInput.tsx

```diff
@@ -49,23 +52,40 @@ const DynamicInput: FunctionComponent<DynamicInputProps & WithDateProps> = ({
     switch (fieldType) {
         case DynamicFormFieldType.dropdown:
             return (
-                <select
-                    {...(rest as any)}
-                    className="form-select optimizedCheckout-form-select"
-                    data-test={`${id}-select`}
-                    id={id}
-                    name={name}
-                    onChange={onChange}
-                    value={value === null ? '' : value}
-                >
-                    {placeholder && <option value="">{placeholder}</option>}
-                    {options &&
-                        options.map(({ label, value: optionValue }) => (
-                            <option key={optionValue} value={optionValue}>
-                                {label}
-                            </option>
-                        ))}
-                </select>
+                <>
+                    <svg
```

Comment:
Could this be an `Icon` component?

Reply:
I didn't know we have a large collection of `Icon` components. 😅

This `svg` has been replaced by the existing `IconChevronDown`.